app/uk/gov/hmrc/datecalculator/services/BankHolidaysService.scala

```diff
@@ -31,32 +30,31 @@ import java.time.format.DateTimeFormatter
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class BankHolidaysService @Inject() (bankHolidaysConnector: BankHolidaysConnector)(implicit ec: ExecutionContext) {
+class BankHolidaysService @Inject() (bankHolidaysConnector: BankHolidaysConnector)(using ExecutionContext) {
 
   private val logger: Logger = Logger(this.getClass)
 
-  def getBankHolidays()(implicit hc: HeaderCarrier): Future[BankHolidays] =
-    bankHolidaysConnector.getBankHolidays().map{ httpResponse =>
-      if (httpResponse.status === OK) {
+  def getBankHolidays()(using hc: HeaderCarrier): Future[BankHolidays] =
```

Comment:
Just for consistency, does this need the hc?

Reply:
oh yeah, I can change it over. The small things just slip through the cracks 😄

Reply:
fixed [here](https://github.com/hmrc/date-calculator/pull/17/commits/d43ef9ce930b0f314bf79c842493444eeacdd550)